.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
+<!--
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>Your code examples should be…
+>   …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the issue itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+-->
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Actual behavior
+<!-- A clear and concise description of what happened instead. -->
+
+## Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+# Environment:
+<!-- (please complete the following information) -->
+
+## Java/JDK
+<!-- `java -version` -->
+
+## Build tool version
+<!-- Remove unused sections -->
+
+### Gradle
+<!-- `gradlew --version` -->
+
+### Apache Maven
+<!-- `mvn --version` -->
+
+## Operating System
+<!--  Linux, Windows, Mac etc. -->
+
+## IDE
+<!--  IntelliJ, Eclipse etc. -->
+
+## Build-tool dependencies used
+
+### Gradle/Grails
+    testImplementation 'org.spockframework:spock-core:2.0-groovy-3.0'
+
+### Apache Maven
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-core</artifactId>
+      <version>2.0-groovy-3.0</version>
+        <scope>test</scope>
+    </dependency>
+## Additional context
```

Comment:
I suggested to remove one of the two blank lines, not both. :-)

Reply:
I just applied your suggestion without modifying it...

Reply:
Hm, then this is a GitHub bug it seems.